app/views/applications/applications.scala.html

```diff
@@ -139,12 +141,12 @@ <h1>Applications</h1>
                     <div class="grid-layout__column--1-3 hard--right">
                         <label class="form-label bold" for="filter">Filter by Environment
                             <select class="form-control input-select input-select--large"
-                            data-datatable-filter="applications-table"
-                            id="environment"
-                            name="environment"
-                            value="">
-                                <option id="default-environment" value="PRODUCTION" data-api-name="PRODUCTION">Production</option>
-                                <option value="SANDBOX" data-api-name="SANDBOX">Sandbox</option>
+                                    data-datatable-filter="applications-table"
+                                    id="environment"
+                                    name="environment"
+                                    onchange="this.form.submit()">
+                                <option @if(environment == Some("PRODUCTION")){selected} id="default-environment" value="PRODUCTION" data-api-name="PRODUCTION">Production</option>
```

Comment:
I think this should say if environment == None || environment == Some("Production"))